.ci/lint-shell.sh

```diff
@@ -11,4 +11,8 @@ docker run \
   --rm \
   koalaman/shellcheck hooks/*
 
+EXIT_CODE=$?
+
 popd >/dev/null
+
+exit $EXIT_CODE
```

Comment:
Same here

Reply:
Answer is same as before, we do this to catch the exit code of the tests and return them as the exit code of the script, so that GitHub actions knows if the tests failed.